quickstarts/kubernetes/google-kubernetes-engine/alerts/HighCPU.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: High Disk Usage
+
+details: |+
+  This alert is triggered when the disk usage is around 100% for at least 15 minutes.
+
+type: STATIC
+nrql:
+  query: "SELECT max(cpuPercent) FROM SystemSample WHERE clusterName LIKE '%'"
```

Comment:
@EmilHammarstrand I think `facet clusterName` should be considered here as this alert could be applied to multiple clusters.

Reply:
I agree, missed that! Thanks @bpschmitt